documentation/dsl.md

```diff
@@ -214,7 +214,7 @@ SonicWeave doesn't have percentages so the CSS color `hsl(120, 60%, 70%)` is spe
 
 ## Code comments
 
-Anything after two slashes (`//`) is ignored until the end of the line. Everything after a slash and an asterisk (`/*`) is ignored until a corresponding pair (`*/`) is encountered.
+Anything after two slashes (`//`) is ignored until the end of the line. Everything after a slash and an asterisk (`/*`) is ignored until a corresponding pair (`*/`) is encountered. (This means you cannot nest multi-line comments, as in C and javascript.)
```

Comment:
I think I've capitalized it *JavaScript* in the docs. Would be nice to have consistency. (Lest we say *ECMAScript* in order to be accurate but obscure.)

Reply:
ah right. ive usually seen javascript in all lowercase when mentioned formally, as suggested by the abbreviation "js".